src/react-apps/applications/runtime/src/utils/language.ts

```diff
@@ -0,0 +1,83 @@
+import { ComponentTypes } from '../components';
```

Comment:
Is this util necessary? Is it used anywhere in runtime? Are you sure you didn't mean to edit the ux-editor util?

Reply:
deleted the file